app/konnect/gateway-manager/provision-cloud-gateway.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Provision a Dedicated Cloud Gateway
+---
+
+This guide explains how to provision a [Dedicated Cloud Gateway](/konnect/dedicated-cloud-gateways/) and scale the data plane nodes in {{site.konnect_short_name}}.
+
+## Prerequisites
+
+* A network configured in {{site.konnect_short_name}} <!--I feel like a link is missing here, but I can't find a good doc to link to-->
+
+## Provision your fully-managed data plane nodes
+
+{% navtabs %}
+{% navtab UI %}
+1. From [Gateway Manager](https://cloud.konghq.com/gateway-manager) in the navigation menu, click the **New Control Plane** menu and select **{{site.base_gateway}}**.
+
+1. In the Create a Control Plane dialog, enter a name for your cloud gateway and click **Dedicated Cloud Instances**.
+
+1. Configure any labels as needed, and click **Next Step**. 
+    
+	You will be redirected the provisioning dialog for your newly created Dedicated Cloud Gateway. The cloud gateway is only created, it still needs to be provisioned and a data plane node needs to be added.
+
+1. To add a data plane node to your Dedicated Cloud Gateway, do the following in the Create a Data Plane Node dialog:
```

Comment:
```suggestion
1. Add a data plane node to your Dedicated Cloud Gateway:
```
The "Create a Data Plane Node dialog" pops up automatically, so we shouldn't need to reorient them.